.github/workflows/test.yml

```diff
@@ -101,6 +104,75 @@ jobs:
 
           skopeo copy oci-archive:dotnet-aspnet.tar docker-daemon:${{ env.aspnet-image-name }}
 
+      - name: Generate SBOM for the .NET runtime deps container image
+        run: |
+          set -x
+          archs=`cat rockcraft.*.dotnet-deps-7.0-${{ matrix.ubuntu-release }}.yaml | shyaml get-value platforms | shyaml keys`
+          buildx_platforms="linux/$(echo ${archs} | sed 's/ /,linux\//g')"
+
+          docker buildx build \
+            --platform=${buildx_platforms} \
+            --output type=oci,dest=dotnet-deps-sbom.tar \
+            -f dotnet-deps/Dockerfile.${{ matrix.ubuntu-release }} \
+            --target sbom-prep \
+            dotnet-deps
+
+          for arch in $archs; do
+            skopeo copy --override-arch "$arch" oci-archive:dotnet-deps-sbom.tar oci-archive:"dotnet-deps-sbom.$arch.tar"
+            syft packages \
+              -o spdx-json \
+              --name "${{ env.runtime-deps-image-name }}" \
+              oci-archive:"dotnet-deps-sbom.$arch.tar" > "dotnet-deps-${{ matrix.ubuntu-release }}.$arch.sbom.json"
```

Comment:
please use the same naming convention as for the ubuntu base image: `<imgName>-oci-<arch>-root.sbom.spdx`

Reply:
Named as `<imgName>-<release>-oci-<arch>-root.sbom.spdx` to include the release information.